torchgeo/datamodules/fair1m.py

```diff
@@ -5,9 +5,33 @@
 
 from typing import Any
 
+import torch
+from torch import Tensor
+
 from ..datasets import FAIR1M
 from .geo import NonGeoDataModule
-from .utils import dataset_split
+
+
+def collate_fn(batch: list[dict[str, Tensor]]) -> dict[str, Any]:
```

Comment:
How is this different from unbind_samples?

Reply:
Because of the torch.stack on line 27